reactioncommerce/reaction-next-starterkit
# feat: add to cart button on PDP

_This takes care of the UI portion fo the ticket. The data portion needs some more GraphQL work to be done before it can be done (#95)._

Adds `Add to Cart` button, with quantity chooser, to the PDP.

Uses `uiStore` to read data pertaining to the currently selected variant / option.

## TESTING
- Uncomment the appropriate code in the `handleOnClick` function in `ProductDetailAddToCart.js`
- Navigate to a product page
- Use the quantity chooser to choose a quantity of item
- Press the "add to cart" button
- In your console, see that the correct amount of items was "added/logged"
